gptcli/session.py

```diff
@@ -63,12 +67,14 @@ def __init__(self, message: str):
 COMMAND_QUIT = (":quit", ":q")
 COMMAND_RERUN = (":rerun", ":r")
 COMMAND_HELP = (":help", ":h", ":?")
+COMMAND_BACK_REGEX = r":b(?:ack)? (\d{1,3})"
```

Comment:
any reason to limit to 999 messages?
```suggestion
COMMAND_BACK_REGEX = r":b(?:ack)? (\d+)"
```

Reply:
Good catch, no there is no reason to keep this limit.